mne_bids/mne_bids.py

```diff
@@ -203,12 +203,15 @@ def _participants_tsv(raw, subject_id, group, fname, verbose):
         # determine the age of the participant
         age = subject_info.get('birthday', None)
         meas_date = raw.info.get('meas_date', None)
+        if isinstance(meas_date, tuple):
+            meas_date = meas_date[0]
+        elif not isinstance(meas_date, tuple) and meas_date is not None:
+            raise ValueError('\n\nUnexpected meas_date: "{}" of type "{}"\n\n'
+                             ''.format(meas_date, type(meas_date)))
```

Comment:
can this actually happen?

Reply:
yes it can! This is my desperate WIP attempt to fix circleci ... because everything runs clean on my machine and I couldn't figure out the error. But now I caught it:

```python
ile "/home/circleci/project/mne_bids/mne_bids.py", line 210, in _participants_tsv
    ''.format(meas_date, type(meas_date)))
ValueError: 

Unexpected meas_date: "[1239115175     618271]" of type "<class 'numpy.ndarray'>"




computation time summary:
	- convert_ds000117.py: 3.2
```

Apparently the PR in MNE-Python to form everything into tuple wasn't as thorough? Or are we somehow not using master for circleci?

Reply:
my god yes ... we are using mne 0.16 for circle ci!
https://github.com/mne-tools/mne-bids/blob/21a46ef2e4370635b4958a28f46deb3c902fd2c8/environment.yml#L33

Reply:
see @sappelhoff sometimes conservative approach pays off ;)

Let me know when it's fixed